eng/pipelines/templates/stages/archetype-java-release.yml

```diff
@@ -165,14 +160,9 @@ stages:
                   deploy:
                     steps:
                       - checkout: self
-                      - template: /eng/pipelines/templates/steps/stage-artifacts.yml
-                        parameters:
-                          SourceFolder: ${{parameters.ArtifactName}}-signed
-                          TargetFolder: ${{artifact.safeName}}
-                          PackageName: ${{artifact.name}}
                       - template: /eng/common/pipelines/templates/steps/docs-metadata-release.yml
                         parameters:
-                          ArtifactLocation: $(Pipeline.Workspace)/${{artifact.safeName}}
+                          ArtifactLocation: ${{parameters.ArtifactName}}-signed/${{artifact.groupId}}/${{artifact.name}}
```

Comment:
Shouldn't this also be `$(Pipeline.Workspace)/${{parameters.ArtifactName}}-signed/${{artifact.groupId}}/${{artifact.name}}`?

Reply:
Fixed.

Reply:
Probably would have worked anyway but consistency is good.